### ISSUE TITLE
Manual tagging when face not detected

### ISSUE THREAD

**Issue by BurningTheSky** [nextcloud/recognize#611]
### Describe the feature you'd like to request

When a face isn't detected in a photo (main example I've seen is when 2 people have faces close together) it would be good to be able to create a manual tag to include the picture.

### Describe the solution you'd like

Ideal solution would be to frame out someones face, like in digikam, but I think creating a manual tag in a photo sidebar might be more feasible? 

### Describe alternatives you've considered

Looked to see if I could manually tag, but either I missed it or can't do it

**Comment by 2nOrderEDO** [comment]
Yes please! It would be super cool if this additional user input can be used for helping clustering and/or better recognize people who are often in our pictures.

I will also be nice if there is a way to show over the screen the detection windows over the faces. Many times a face would be recognized, but nothing will be done with it. If the user can click on this window and assign a person, that would be very helpful.

**Comment by XotEmBotZ** [comment]
What is the status on this?

**Comment by marcelklehr** [comment]
@XotEmBotZ This is not being worked on currently. We're planning to do a revamp of recognize some time soon which should also improve face recognition.

**Comment by smirkybg** [comment]
Genuinely interested in this. I've been using Recognize from almost an year now and this feature seems to be severely needed, since it's an alternative when the models aren't good enough.

**Comment by kiber-io** [comment]
It's really strange that such a seemingly basic feature is missing :(
Recognize failed to detect faces in a lot of photos, even though the faces literally take up most of the picture.
And now it turns out I can't even manually link these photos to people.

**Comment by marcelklehr** [comment]
Please refrain from commenting if you have no substantial contribution to make other than "I want this, too" 💙

**Comment by skylord123** [comment]
A more advanced model is nice but still wont solve the issue entirely. I want to be able to tag people wearing masks, costumes, etc. Manual tagging for these situations would still be required.

Personally I would want something like FB that would allow me to tag someone even if their face isn't shown. Not sure how feasible that is but it would be really powerful and make filtering images by person much more useful. In it's current form many of my images that contain someone do not show up under that person due to these edge cases.

**Comment by csidirop** [comment]
100% what skylord123 imagines. 

Also additionally, although I think it goes hand-in-hand with the feature request, that **unmarking** of tagged people would be very important! In my case I have a lot of false matches.
